webapp/src/app/components/sidenav/sidenav.component.ts

```diff
@@ -14,7 +14,7 @@ import {MatTabChangeEvent} from '@angular/material';
 })
 export class SidenavComponent implements OnInit {
 	
-	activeTab = 0;
+	activeTab = EditorView.Layers;
```

Comment:
If `activeTab` is changed to EditorView then it is the same as `currentView`. In this case the `currentView` variable could be removed

Reply:
`currentView` is in fact unused. Fixed by removal in fdd25bafcbf6f475002ed4916aa8757f3d4bd3cc.